catalyst_voices/packages/internal/catalyst_voices_localization/lib/l10n/intl_en.arb

```diff
@@ -967,5 +967,30 @@
   "reviewRegistrationTransaction": "Review registration transaction",
   "@reviewRegistrationTransaction": {
     "description": "A button label to review the registration transaction in wallet detail panel."
+  },
+  "guidance": "{guidance, select, mandatory{Mandatory} education{Education} other{Tips}}",
```

Comment:
i think this kind of select is easy to forget about and make bugs if we ever change guidance type or add new type.

consider making separate names for each guidance type